cli/main.c

```diff
@@ -1616,6 +1616,12 @@ static int fw_read(int argc, char **argv)
 	else
 		inf = cfg.inactive ? sum->img.inactive : sum->img.active;
 
+	if (!inf || !inf->valid) {
+		ret = -1;
+		fprintf(stderr, "fw_part_summary: No valid image for this partition!\n");
```

Comment:
Please remove the fw_part_summary from the message, usually we specify the function name in switchtec_perror() in error. It's not necessary in this case.

Reply:
Updated, please review.